core/modules/qdisp/QdispPool.h

```diff
@@ -152,8 +151,8 @@ class QdispPool {
 
 private:
     /// The default priority queue is meant for pool control commands.
-    PriorityQueue::Ptr _prQueue = std::make_shared<PriorityQueue>(100,1); // default (lowest) priority.
-    util::ThreadPool::Ptr _pool{util::ThreadPool::newThreadPool(30, _prQueue)};
+    PriorityQueue::Ptr _prQueue = std::make_shared<PriorityQueue>(100, 1, 5); // default (lowest) priority.
+    util::ThreadPool::Ptr _pool{util::ThreadPool::newThreadPool(1200, _prQueue)};
 };
 
```

Comment:
OK what's the difference between comment with three slashes and ones with only two?